tests/stick_test_data.py

```
@@ -0,0 +1,1381 @@
+from datetime import UTC, datetime, timedelta
```

Ruff (0.11.9)

Missing docstring in public module

(D100)